README.md

```diff
@@ -65,4 +66,4 @@ If you want to view the production build locally, run:
 gatsby serve
 ```
 
-Once this starts, you can view your site at http://localhost:9000.
\ No newline at end of file
+Once this starts, you can view your site at http://localhost:9000.
```

Comment:
This is results from "No newline at end of file". How to we handle this? Should I try to use "git add -p" (see https://stackoverflow.com/questions/11414392/git-diff-thinks-the-last-line-of-the-file-has-been-moved-after-the-end-of-file) or always append a new line?

Reply:
I like a newline at the end of the file, also see second answer in https://stackoverflow.com/questions/11414392/git-diff-thinks-the-last-line-of-the-file-has-been-moved-after-the-end-of-file

Reply:
Ok, then we leave this file like it is now, i.e. with an `\n` at the end. Should then document this somewhere?

Reply:
Maybe in the README.md itself, like 'Every file ends with a newline.' in a new section 'Conventions'. I think there are more files that do not end with a newline at the moment (e.g. gatsby-config.js, package.json ...) 🤔 Should we convert them all at once?

Reply:
As by coincidence - we yesterday had [formatting fixes in metafacture-core](https://github.com/metafacture/metafacture-core/pull/388) using a `.editorconfig`. We could reuse it here. With `gradle` it is possible to autoformat the code at command line (@fsteeg right?), but it seems that this is not directly possible and needs some tweak with npm - maybe we don't need this because there are only a few files in question?

Reply:
Maybe we could add newlines everytime we need to change a file anyway.

Reply:
I'm good with applying code style when changing files. For future reference I added
https://github.com/skohub-io/skohub-blog/pull/6/commits/d0b56ee444f4541564b3cec820b5a69185e5a534#diff-0947e2727d6bad8cd0ac4122f5314bb5b04e337393075bc4b5ef143b17fcbd5b which can be applied to IDE and editors to have a consistent code style for every new file.